state/txn_test.go

```diff
@@ -73,6 +77,46 @@ func TestSnapshotUpdateData(t *testing.T) {
 	assert.Equal(t, hash1, txn.GetState(addr1, hash1))
 }
 
+func TestGetDumpTree(t *testing.T) {
+	txn := newTestTxn(defaultPreState)
+	txn.SetState(addr1, hash1, hash2)
+	txn.SetState(addr2, hash2, hash1)
+
+	dump := &Dump{
+		Accounts: make(map[types.Address]DumpAccount),
+	}
+	opts := &DumpInfo{
+		Start:             []byte{},
+		Max:               1,
+		OnlyWithAddresses: false,
+		SkipCode:          false,
+		SkipStorage:       false,
+	}
+
+	nextKey, err := txn.GetDumpTree(dump, opts, false)
+
+	assert.NoError(t, err)
+	assert.Equal(t, addr2.Bytes(), nextKey)
+	assert.Equal(t, 1, len(dump.Accounts))
```

Comment:
```suggestion
	assert.Len(t, dump.Accounts, 1)
```
As a nitpick for slices length assert we can use Len method.